cmd/aro/rp.go

```diff
@@ -61,6 +62,7 @@ func rp(ctx context.Context, log, audit *logrus.Entry) error {
 			"CLUSTER_MDM_NAMESPACE",
 			"MDM_ACCOUNT",
 			"MDM_NAMESPACE",
+			env.OIDCStorageAccountName,
```

Comment:
Why not use `OIDC_STORAGE_ACCOUNT_NAME` to match the other vars in this slice?

Reply:
Since the `OIDC_STORAGE_ACCOUNT_NAME ` is used at multiple places:-
Validation and Fetching env variable. Preferred to have a single source of truth in a const.